src/app/@ansyn/ansyn/modules/menu-items/cases/services/cases.service.ts

```diff
@@ -41,67 +32,41 @@ export class CasesService {
 	queryParamsHelper: QueryParamsHelper = new QueryParamsHelper(this);
 	baseUrl;
 	paginationLimit = 15;
-	queryParamsKeys;
 	latestStoredEntity: any;
 
-	constructor(protected storageService: StorageService,
-				@Inject(casesConfig) public config: ICasesConfig,
-				@Inject(linksConfig) public linksConfig: ILinksConfig,
-				public urlSerializer: UrlSerializer,
-				protected translator: TranslateService,
-				public errorHandlerService: ErrorHandlerService) {
-		this.paginationLimit = this.config.paginationLimit;
-		this.queryParamsKeys = this.config.casesQueryParamsKeys;
-	}
-
 	get defaultCase() {
 		return this.config.defaultCase;
 	}
 
-	getLink(linkId: string) {
-		return this.storageService.get(this.linksConfig.schema, linkId).pipe(
-			map(caseData => {
-				const dilutedCase: IDilutedCase = {
-					state: <ICaseState>caseData.data,
-					creationTime: new Date(),
-					id: linkId,
-					schema: 'link'
-				};
-				return this.parseCase(dilutedCase);
-			}),
-			catchError(() => EMPTY)
-		);
-	}
-
-	get generateQueryParamsViaCase() {
-		return this.queryParamsHelper.generateQueryParamsViaCase.bind(this.queryParamsHelper);
-	}
-
 	get updateCaseViaContext() {
 		return this.queryParamsHelper.updateCaseViaContext.bind(this.queryParamsHelper);
 	}
 
-	loadCases(casesOffset: number = 0): Observable<any> {
-		return this.storageService.getPage<ICasePreview>(this.config.schema, casesOffset, this.paginationLimit)
+	constructor(protected storageService: StorageService,
+				@Inject(casesConfig) public config: ICasesConfig,
+				public errorHandlerService: ErrorHandlerService) {
+		this.paginationLimit = this.config.paginationLimit;
+	}
+
+	loadCases(casesOffset: number = 0, casesType: 'owner' | 'sharedWith' = 'owner'): Observable<any> {
```

Comment:
maybe create a type like CaseClearence for 'owner' and 'sharedWith'

Reply:
already have.